internal/providers/onesignal.provider.go

```
@@ -111,24 +113,29 @@ func (e *OneSignal) SendSms(phone string, content string) error {
 		return reqErr
 	}
 	return nil
 }
 
-func (e *OneSignal) sendRequest(json []byte) error {
+func (e *OneSignal) sendRequest(ctx context.Context, json []byte) error {
+	client := &http.Client{
+		Transport: otelhttp.NewTransport(http.DefaultTransport),
+	}
+
 	req, errReq := http.NewRequest("POST", e.url, bytes.NewBuffer(json))
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)